app/models/atmosphere/virtual_machine_template.rb

```diff
@@ -144,6 +144,12 @@ def perform_delete_in_cloud
       logger.warn("VMT with #{id_at_site} does not exist - continuing")
     end
 
+    # Returns the hourly cost for this template assuming a given VM flavor
+    def get_hourly_cost_for(flavor)
+      incarnation = flavor.flavor_os_families.find_by(os_family: appliance_type.os_family)
+      incarnation && incarnation.hourly_cost
+    end
```

Comment:
Again this kind of execution. I'm not convinced it's going in the right direction. I understand the intent - make sure we do not fail with the infamous 'nil has no method called hourly_cost' exception but, what we get instead, is that the code that calls this method receives false (and it expected a number, I guess). So, we'll probably get an exception elsewhere anyway (it will be something like "cannot compare false to 12.4").